CVE-2023-0386/cvex.yml

```diff
@@ -0,0 +1,5 @@
+blueprint: ubuntu2204
+ubuntu:
+  playbook: ubuntu.yml
+  command:
+    - "cd /usr/local/bin && ./exp"
```

Comment:
It gives me this:

> 2024-11-20 12:33:22,381 - INFO - [ubuntu] Executing 'cd /usr/local/bin && ./exp'...
> 2024-11-20 12:33:22,474 - INFO - [ubuntu] [+] exploit success!
> 
> 2024-11-20 12:33:22,474 - INFO - [ubuntu] exp: mkdir ./ovlcap: Permission denied
> sh: 1: ./ovlcap/upper/file: not found

Why do we see "Permission denied" after "exploit success!"? Is it OK?

Reply:
in my local test without ansible, it gives "exploit success!", followed by: sh: ... (waiting for prompt), i'll look into this